### PR TITLE
Docs: Typo, grammar

### PR DESCRIPTION
**Pull request by michaelKaefer**
| Q             | A
| ------------- | ---
| Branch?       | master for features / 3.4, 4.4 or 5.0 for bug fixes 
| Bug fix?      | yes/no
| New feature?  | yes/no 
| Deprecations? | yes/no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Security/Core/User/UserInterface.php [hunk 1]
```diff
@@ -19,10 +19,10 @@
...
  * and so on.
  *
- * Regardless of how your user are loaded or where they come from (a database,
- * configuration, web service, etc), you will have a class that implements
- * this interface. Objects that implement this interface are created and
- * loaded by different objects that implement UserProviderInterface
+ * Regardless of how your users are loaded or where they come from (a database,
+ * configuration, web service, etc.), you will have a class that implements
+ * this interface. Objects of that class are created and loaded by different
+ * objects that implement UserProviderInterface
```
Comment (OskarStark) [inline src/Symfony/Component/Security/Core/User/UserInterface.php]: classes that ....

?

Comment (chalasr) [reply src/Symfony/Component/Security/Core/User/UserInterface.php]: I would only fix `user` and `etc`, the rest does not sound better than what we have already TBH

Comment (michaelKaefer) [reply src/Symfony/Component/Security/Core/User/UserInterface.php]: Ok, thank you, I changed it back. For me it sounds strange to say that an _object_ implements an interface.

Comment (michaelKaefer) [reply src/Symfony/Component/Security/Core/User/UserInterface.php]: @OskarStark I am not sure how to understand your review..

**Comment by chalasr** [comment]
Good catch, thanks @michaelKaefer.

**Pull request merged**